fission/src/Synthesis.tsx

```diff
@@ -66,6 +66,7 @@ import NewInputSchemeModal from "./ui/modals/configuring/theme-editor/NewInputSc
 import AssignNewSchemeModal from "./ui/modals/configuring/theme-editor/AssignNewSchemeModal.tsx"
 import AnalyticsConsent from "./ui/components/AnalyticsConsent.tsx"
 import PreferencesSystem from "./systems/preferences/PreferencesSystem.ts"
+import APSManagementModal from "./ui/modals/APSManagementModal.tsx"
```

Comment:
Can someone explain to me the difference between different `@` imports and seemingly relative imports? I understand that `@` refers to a module (or the javascript / typescript equivalent. Essentially a directory or collection of files), but we seem to switch between using an `@` and a `./` and I'm not familiar with why.

Reply:
We specify what the @ ones point to in our tsconfig and vite config, and they make it easier to change imports if we were to do a refactor since we’d only need to change there (for the most part). I think we switch between them because our editors auto-import using relative imports and nobody bothers to change it. It might be worth going through everything at some point and standardizing it.

Reply:
I agree, not too big of a deal but would be nice if everything was the same.